app/views/issues/show.html.slim

```diff
@@ -17,9 +17,19 @@
           .follow.pull-right
             - if user_signed_in? && current_user.active?
               = render 'shared/follow_link', issue: @issue
-            .followers-count
-              span Спостерігачів:
-              span.badge.pull-right #{@issue.followers_count}
+              .followers-count
+                span Спостерігачів:
+                span.badge.pull-right #{@issue.followers_count}
+        .votes-wrapper
+          .votes.pull-right
+            - if user_signed_in? && current_user.active?
+              - if @voted.empty?
```

Comment:
@Kmalynss I don't think so, we will have the result of true every time we call "current_user.active?", we will have false only if user is banned.